R/igv.R

```diff
@@ -13,7 +13,7 @@ handleCategory <- function(x) {
     categories <- lapply(getNodeSet(x, "Category"), handleCategory)
     children <- do.call(rbind, categories)
     children$parenet <- name
-    resources <- rtracklayer:::RTLFileList(lapply(paths, FileForFormat))
+    resources <- rtracklayer:::BiocFileList(lapply(paths, FileForFormat))
```

Comment:
We should probably drop the `rtracklayer:::`. Probably wasn't supposed to be there in the first place.